scripts/import_from_ena.py

```diff
@@ -27,22 +33,57 @@
 DATA_SOURCES = ['fastq', 'sra', 'cram_index']
 DATA_TYPES = ['ftp', 'galaxy', 'aspera']
 
+logger = logging.getLogger(__name__)
+logging.basicConfig(format='%(asctime)s\t%(levelname)s:\t%(name)s line %(lineno)s\t%(message)s', level=logging.INFO)
+# suppress logging information from elasticsearch
+logging.getLogger('elasticsearch').setLevel(logging.WARNING)
 
-def main():
+
+@click.command()
+@click.option(
+    '--es_hosts',
+    default="wp-np3-e2;wp-np3-e3",
```

Comment:
We have default addresses for staging, fallback and production ES in constants module, so it's better to use it from there

Reply:
done in 9d53f80